package.json

```diff
@@ -12,9 +12,9 @@
     "@babel/node": "^7.13.0",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/preset-env": "^7.13.5",
-    "@hathor/wallet-lib": "^0.42.1",
-    "express-validator": "^6.10.0",
```

Comment:
The IDE changed the position of the `express-validator` lib on its dependency auto-sorting.